### PR TITLE
replace webr extension; upkeep

### PR DESCRIPTION
**Pull request by pawelru**
- replace [`r-wasm/quarto-live`](https://github.com/r-wasm/quarto-live) with [`coatless/webr`](https://github.com/coatless/quarto-webr) - close https://github.com/insightsengineering/tlg-catalog/issues/281
  - remove one extension
  - add another one
  - slightly change css - there is a new div ID
  - updates webr code chunks -> `{webr-r}` into `{webr}`
  - update webr config options in `_quarto.yaml`
  - change the format from `html` into `live-html`
  - `live-html` webr code chunks has access to previous webr code chunks. Therefore we don't have to include "setup" chunks in the webr code chunks. That reduces the amount of code the user sees which is a nice feature.
- Other changes:
  - fixes layout error - close https://github.com/insightsengineering/tlg-catalog/issues/285
  - apply full width
  - apply auto-formatting from VSCode quarto extension
  - add `## Output` - to prepare for extending catalog with cards - so that it would be easier to add new child sections inside Outputs
  - remove Quarto comments (i.e. `<!-- ... --!>`) as these are pushed to the output HTML document
  - add `warning = FALSE, message = FALSE` to the startup code chunks to hide package startup messages
  - change inline r code chunk (with `webr_code_labels`) to multiline code chunk so that it won't be changed by IDE autoformatters. This was already detected previously and apparently in this case the spaces matter so it would prevent auto-destroys in the future
  - review all existing skips - skip strict, skip tests, skip eval

### CODE REVIEW

File: book/_quarto.yml [hunk 1]
```diff
@@ -41,16 +44,19 @@ execute:
...
 engine: knitr
+
 webr:
-  channel-type: "post-message"
-  packages: ["tern", "random.cdisc.data", "dplyr"]
-  show-startup-message: false
-  autoload-packages: false
+  packages:
+    - tern
+    - random.cdisc.data
```
Comment (shajoezhu) [inline book/_quarto.yml:51]: should we also include pharamverseadam

File: book/graphs/other/bwg01.qmd [hunk 2]
```diff
@@ -79,6 +77,15 @@ bp_annos <- function(bp, color, annos = 1) {
...
 
+```{r include = FALSE}
+webr_code_labels <- c("setup")
+```
+
+{{< include ../../_utils/webr_no_include.qmd >}}
+
+## Output
+
+:::::::::::: panel-tabset
```
Comment (shajoezhu) [inline book/graphs/other/bwg01.qmd:88]: hi @pawelru , the number of ":" varies from template to template, I was wondering why is that? what does the ":" do?

**Review by shajoezhu — APPROVED** [review]
thanks @pawelru , lgtm! approve to unblock merge

File: book/_quarto.yml (hunk 1 above)
Comment (pawelru) [reply book/_quarto.yml:51]: Currently `pharmaverseadam` is not used at all. If we want to change this then that probably had to be done as a separate PR not to explode with this one

File: book/graphs/other/bwg01.qmd (hunk 2 above)
Comment (pawelru) [reply book/graphs/other/bwg01.qmd:88]: These were added automatically by Quarto extension in VSCode. It remains fully functional but I agree - this is just confusing. I will try to look at this again to see if we can make it more consistent across articles but on the other hand I want to limit these auto-edits in the future.

Comment (pawelru) [reply book/graphs/other/bwg01.qmd:88]: This continue to get auto-updates. When I change it back to a fewer `:` then Quarto ext change it back to more. I think I don't want to go and dig into that. It's not a big deal.

**Pull request merged**